src/dialogs/examples/basic/PopupSignInDialog.ts

```diff
@@ -0,0 +1,51 @@
+import * as builder from "botbuilder";
+import { TriggerActionDialog } from "../../../utils/TriggerActionDialog";
+import { DialogIds } from "../../../utils/DialogIds";
+import { DialogMatches } from "../../../utils/DialogMatches";
+import * as config from "config";
+
+export class PopupSignInDialog extends TriggerActionDialog {
+
+    private static async PopUpSignIn(session: builder.Session, args?: any | builder.IDialogResult<any>, next?: (args?: builder.IDialogResult<any>) => void): Promise<void> {
```

Comment:
Nit: TS convention is lowercase method names.

Reply:
Done.